addons/l10n_id/tests/test_qris.py

```diff
@@ -94,13 +93,13 @@ def test_qris_qr_code_generation(self):
             result = self.qris_qr_invoice.with_context({'is_online_qr': True})._generate_qr_code()
             self.assertIsNotNone(result)
 
-            # Confirm that the QR was successfully registered on the invoice.
-            qr_details = self.qris_qr_invoice.l10n_id_qris_invoice_details
-            self.assertNotEqual(qr_details, False)
+            # Confirm that a transaction should be registered on the invoice
+            qr_details = self.qris_qr_invoice.l10n_id_qris_transactions
+            self.assertEqual(len(self.qris_qr_invoice.l10n_id_qris_transactions), 1)
 
             qris_data = self.success_qris_get['data']
-            self.assertEqual(qr_details[-1]['qris_invoice_id'], qris_data['qris_invoiceid'])
-            self.assertEqual(qr_details[-1]['qris_content'], qris_data['qris_content'])
+            self.assertEqual(qr_details['qris_invoice_id'], qris_data['qris_invoiceid'])
+            self.assertEqual(qr_details['qris_content'], qris_data['qris_content'])
```

Comment:
Please access the field on the transaction model using qr_details.xxx instead
Most standard, and easier to understand what's happening (applies to all tests)